R/use_python.R

```diff
@@ -147,6 +147,17 @@ use_condaenv <- function(condaenv = NULL, conda = "auto", required = NULL) {
 
   required <- required %||% use_python_required()
 
+  if (grepl("[/\\]", condaenv) &&
+      grepl("^python[0-9.]*(exe)?", basename(condaenv))) {
+    # path to python binary provided
+    python <- condaenv
+    if (!is_conda_python(python))
+      warning("path supplied to use_condaenv() is not a conda python:\n\t",
+              path)
+    use_python(python, required = required)
```

Comment:
I wonder if we should return the value of `use_python()` (here and elsewhere in the function)